ruby/vampires.rb

```diff
@@ -0,0 +1,103 @@
+puts "How many employees will be processed?"
+n = gets.chomp.to_i
+
+while n > 0
+  puts "Thank you for taking our new employee survey!"
+  
+  # Questions to collect data from the employee.
+
+  puts "What is your name?"
+  name = gets.chomp
+  
+  puts "How old are you, or how old will you be turning this year?"
+  age = gets.chomp.to_i
+  
+  puts "What year were you born?"
+  birth_year = gets.chomp.to_i
+  by_age = 2017 - birth_year
```

Comment:
Check out the Ruby DateTime class, instead of hardcoding `2017` you can use the current year that will change dynamically.